tests/tests_integration/test_api/test_files.py

```diff
@@ -100,3 +135,12 @@ def test_retrieve_file_with_labels(self, new_file_with_label):
         res = COGNITE_CLIENT.files.retrieve(id=file.id)
         assert len(res.labels) == 1
         assert res.labels[0].external_id == label_external_id
+
+    def test_filter_file_on_geoLocation(self, new_file_with_geoLocation, mock_geo_location):
+        file = new_file_with_geoLocation
```

Comment:
This line can be removed, I think? I don't see it being used elsewhere

Reply:
It is used in the assertion at the bottom. But it doesn't have to be its own line, so I moved it down to be inline :)